core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -0,0 +1,452 @@
+package cats
+package data
+
+import cats.functor.Bifunctor
+import cats.instances.either._
+import cats.syntax.either._
+
+/**
+ * Transformer for `Either`, allowing the effect of an arbitrary type constructor `F` to be combined with the
+ * fail-fast effect of `Either`.
+ *
+ * `EitherT[F, A, B]` wraps a value of type `F[Either[A, B]]`. An `F[C]` can be lifted in to `EitherT[F, A, C]` via `EitherT.right`,
+ * and lifted in to a `EitherT[F, C, B]` via `EitherT.left`.
+ */
+final case class EitherT[F[_], A, B](value: F[Either[A, B]]) {
```

Comment:
Hm I'm getting a lot of `EitherT.scala:253 bridge generated for method compare: (x: cats.data.EitherT[F, L, A], y: cats.data.EitherT[F, L, A])Int in trait EitherTOrder which overrides method compare: (x: A, y: A)Int in trait Order clashes with definition of member itself; both have erased type (x: Object, y: Object)Int` and similar for other methods (11 such occurences)